# Service to set digital outputs on/off
int64 state  #1-on; 0-off
string DIO_num  #digital IO number
---
